libs/core/langchain_core/language_models/base.py

```diff
@@ -113,7 +113,7 @@ def _get_verbosity() -> bool:
 
 
 class BaseLanguageModel(
-    RunnableSerializable[LanguageModelInput, LanguageModelOutputVar], ABC
+    RunnableSerializable[LanguageModelInput, LanguageModelOutputVar], BaseModel, ABC
```

Comment:
`RunnableSerializable` is in `runtime-evaluated-base-classes` but ruff doesn't seem to see it...
Also ruff could see that `BaseLanguageModel` itself is in `runtime-evaluated-base-classes`...